coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json

```diff
@@ -28,16 +28,19 @@
V0z\" fill=\"none\"/><path d=\"M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z\"/></svg> </td><td align=\"left\" class=\"quick-help-description\"> <h5>The above link does not work</h5> <p>Copy/Paste this URL into your Browser:<br/><a href=\"https://catenax-dev003-app-onboarding.azurewebsites.net/?company={companyname}&user={nameCreatedBy}\">https://catenax-dev003-app-onboarding.azurewebsites.net/?company={companyname}&user={nameCreatedBy}</a></p></td></tr><tr> <td align=\"left\" class=\"question-mark-icon\"> <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 0 24 24\" width=\"24px\" fill=\"#000000\"><path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z\"/></svg> </td><td align=\"left\" class=\"quick-help-description\"> <h5>I did not receive the email with the password</h5> <p><a href=\"https://catenax-dev003-app-portal.azurewebsites.net/home/help\">Click here to resend the password</a></p></td></tr><tr> <td align=\"left\" class=\"question-mark-icon\"> <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 0 24 24\" width=\"24px\" fill=\"#000000\"><path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z\"/></svg> </td><td align=\"left\" class=\"quick-help-description\"> <h5>I need more information and help</h5> <p>Copy/Paste this URL into your Browser:<br/><a href=\"https://catenax-dev003-app-portal.azurewebsites.net/home/help\">Checkout our help section or contact support@catena-x.com</a></p></td></tr></tbody> </table> </td></tr></table> </td></tr></table> <div class=\"footer\"> <table role=\"presentation\" border=\"0\" cellpadding=\"0\" cellspacing=\"0\"> <tr> <td class=\"content-block\"> <h3>Catena-x Automotive Network e.V.i.G. </h3> <span class=\"footer-span\"> c/o IFOK GmBH </span> </td></tr><tr> <td class=\"content-block footer-plain-text padding-bottom\"> <span class=\"footer-span\"> ReinhardtstraBe 58 <br/> 10117 Berlin <br/> E-Mail: <a href=\"mailto:info@catena-x.net?subject=Register Mail Reply&body=Please describe your request in this mail body\">info@catena-x.net</a> </span> </td></tr><tr> <td class=\"content-block footer-plain-text padding-bottom\"> <span class=\"footer-span\"> Vertretungsberechtigter Vorstand: Oliver Ganser (Vorstandsvorsitzender): </span> </td></tr></table> </div></div></td><td>&nbsp;</td></tr></table></body></html>",
```

Comment:
Hi Fatih, the url is environment dependent, I don't think it's a good idea to set it to https://catenax-dev003-app-onboarding.azurewebsites.net, you should use the url mailParameter which is defined in the Businesslogic as "BasePortalAddress")}/?company={realm}". Also in this context, you shouldn't need to provide companyname and nameCreatedBy as parameters in the url.

Reply:
I made those changes and pushed again. Could you please take a look new changes ?

Reply:
I updated that variables regarding informations

Reply:
Looks good, but companyname should not be twice passed in the url, that won't work. It makes sense to remove companyname from the url as it's already passed in the url variable (BasePortalAddress")}/?company={realm}").